draft-ietf-quic-tls.md

```diff
@@ -780,9 +780,12 @@ connection ID in the client's first Initial packet (see {{initial-secrets}}).
 This provides protection against off-path attackers and robustness against QUIC
 version unaware middleboxes, but not against on-path attackers.
 
-All ciphersuites currently defined for TLS 1.3 - and therefore QUIC - have a
-16-byte authentication tag and produce an output 16 bytes larger than their
-input.
+QUIC can use any of the ciphersuites defined in {{!TLS13}} with the exception of
```

Comment:
use normative language perhaps?
```suggestion
QUIC can use any of the ciphersuites defined in {{!TLS13}} but MUST NOT use
```

Reply:
Yeah my understanding was that was the conclusion in #2019, but I might be incorrect.

Reply:
I'm happy to say that it is not specified and leave it at that.  The effect is the same.

Reply:
@kazuho @martinthomson : I trust your judgement here, so I won't push. It's still not clear to me why disallowing is an issue. Don't mind me though.